code/ui/components/src/new/Input/Input.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Input } from './Input';
+
+const meta: Meta<typeof Input> = {
+  title: 'Input',
+  component: Input,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof Input>;
+
+export const Base: Story = {
```

Comment:
Consider adding more diverse storybook examples to cover different use cases and edge cases. [medium]

Reply:
Interesting one. I could do that.